build/storage/tests/it/ptf_tests/system_tools/test_platform.py

```diff
@@ -96,62 +155,171 @@ def _install_docker(self):
     def check_system_setup(self):
         """Overwrite this method in specific platform if you don't want check all setup"""
         if not self._is_virtualization():
-            raise Exception("Virtualization is not setting properly")
+            raise DependenciesException("Virtualization is not setting properly")
         if not self._is_kvm():
-            raise Exception("KVM is not setting properly")
+            raise DependenciesException("KVM is not setting properly")
         if not self.pms:
-            raise Exception("Packet manager is not setting properly")
+            raise DependenciesException("Packet manager is not setting properly")
         if not self._is_qemu():
-            raise Exception("QUEMU is not setting properly")
+            raise DependenciesException("QUEMU is not setting properly")
         if not self._set_security_policies():
-            raise Exception("Security polices is not setting properly")
+            raise DependenciesException("Security polices is not setting properly")
         if not self._is_docker():
-            raise Exception("Docker is not setting properly")
+            raise DependenciesException("Docker is not setting properly")
 
-    # TODO: after testing restore settings
     def set_system_setup(self):
```

Comment:
Provide a more meaningful name and description - system_setup may be too generic. If on the other hand this method is supposed to cover all system setup related tasks, then provide a detailed description what it does.

Reply:
deleted - user should themselves set properly env